test/renderers/array-renderer.test.ts

```diff
@@ -93,7 +124,38 @@ test('generate array child control w/o data', t => {
     };
     const data = {};
     renderer.setDataService(new DataService(data));
-    renderer.setDataSchema(schema);
+    renderer.setDataModel({
+  schema: {type: 'object', properties: {test: {type: 'array', items: {
```

Comment:
Why isn't the schema used we already have in scope? Applies to other tests as well, check for unused local vars.